frontend/src/__tests__/utils/form-actions.test.ts

```
@@ -483,4 +485,102 @@ describe('form-actions', () => {
 
     expect(actualOrder).toEqual(expectedOrder);
   });
+
+  describe('setTemplateToSubmitted', () => {
+    test('submitTemplate successfully', async () => {
+      const responseData = {
+        id: 'id',
+        templateType: 'NHS_APP',
+        templateStatus: 'NOT_YET_SUBMITTED',
+        name: 'name',
+        message: 'message',
+        createdAt: '2025-01-13T10:19:25.579Z',
+        updatedAt: '2025-01-13T10:19:25.579Z',
+      } satisfies TemplateDto;
+
+      mockedTemplateClient.submitTemplate.mockResolvedValueOnce({
+        data: responseData,
+      });
+
+      const response = await setTemplateToSubmitted('id');
+
+      expect(mockedTemplateClient.submitTemplate).toHaveBeenCalledWith(
+        'id',
+        'token'
+      );
+
+      expect(response).toEqual(responseData);
+    });
+
+    test('submitTemplate - should thrown error when saving unexpectedly fails', async () => {
```

Typo here and in the next - 'should throw an error'